onnxscript/function_libs/torch_lib/ops/linalg.py

```diff
@@ -44,10 +44,28 @@ def aten_linalg_cond(self: TensorType, p: Optional[float] = None) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_linalg_cross(self: TensorType, other: TensorType, dim: int = -1) -> TensorType:
-    """linalg_cross(Tensor self, Tensor other, *, int dim=-1) -> Tensor"""
+@torch_op("aten::linalg_cross")
+def aten_linalg_cross(self: TTensor, other: TTensor, dim: int = -1) -> TTensor:
 
-    raise NotImplementedError()
+    zero = op.Constant(value_ints=[0])
+    one = op.Constant(value_ints=[1])
+    two = op.Constant(value_ints=[2])
+    three = op.Constant(value_ints=[3])
+    axes = op.Expand(dim, op.Constant(value_ints=[1]))
+
+    # Reference https://en.wikipedia.org/w/index.php?title=Cross_product&oldid=1143125073
+    a1 = op.Slice(self, zero, one, axes)
+    a2 = op.Slice(self, one, two, axes)
+    a3 = op.Slice(self, two, three, axes)
```

Comment:
Is that axis expected to have a dimension exactly 3, or can it have more? If it is exactly 3, can we rewrite the above three as
```py
   a1, a2, a3 = op.Split(self, axis=dim, num_outputs=3)
```

Reply:
I think so. @shubhambhokare1 feel free to update my original implementation. Thanks!